Updates monitoring in installs; rewrite of ICP install (#786)
knative/docs: install/Knative-with-ICP.md

   ```shell
   curl -L https://github.com/knative/serving/releases/download/v0.3.0/monitoring.yaml \
     | sed 's/LoadBalancer/NodePort/' \
     | kubectl apply --filename -
   ```

1. Monitor the Knative components until all of the components show a
   `STATUS` of `Running`:
    ```bash
    kubectl get pods --namespace knative-serving
    kubectl get pods --namespace knative-build
    kubectl get pods --namespace knative-eventing
    kubectl get pods --namespace knative-sources
    kubectl get pods --namespace knative-monitoring
    ```

   > Note: Instead of rerunning the command, you can add `--watch` to the above
   > command to view the component's status updates in real time. Use CTRL+C to
   > exit watch mode.

Now you can deploy an app to your newly created Knative cluster.

## What's next

Now that your cluster has Knative installed, you can see what Knative has to
offer.

To deploy your first app with Knative, follow the step-by-step
[Getting Started with Knative App Deployment](getting-started-knative-app.md)
guide.

> **Note**: When looking up the IP address to use for accessing your app, you need
  the address used for ICP. The following command looks up the value to

>   use for the {IP_ADDRESS} placeholder used in the samples:

Reviewer: ```suggestion
  use for the {IP_ADDRESS} placeholder in the samples:
```

  ```shell
  echo $(ICP cluster ip):$(kubectl get svc istio-ingressgateway --namespace istio-system \
  --output 'jsonpath={.spec.ports[?(@.port==80)].nodePort}')
  ```

> To get started with Knative Eventing, pick one of the

Reviewer: ```suggestion
To get started with Knative Eventing, walk through one of the
```

> [Eventing Samples](../eventing/samples/) to walk through.

Reviewer: ```suggestion
[Eventing Samples](../eventing/samples/).
```
